src/quo2/components/list_items/dapp/style.cljs

```diff
@@ -0,0 +1,41 @@
+(ns quo2.components.list-items.dapp.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(defn container-outer
+  [{:keys [state blur? customization-color theme]}]
+  {:flex               1
+   :padding-horizontal 12
+   :padding-vertical   8
+   :border-radius      12
+   :background-color   (cond
+                         (and (= state :pressed) (= theme :dark) blur?)
+                         colors/white-opa-5
+
+                         (= state :pressed)
+                         (colors/custom-color customization-color 50 5)
```

Comment:
Yea but you probably have to have this themed too as it is 60 for dark.
E.g
(Colors/theme-colors 
(Colors/custom-color ... 50 5)
(Colors/custom-color ... 60 5)
theme)

Reply:
Yes, we should use 60 variants for the dark theme, but for some reason, in [Figma](https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=25947-280499&mode=design&t=3O82IWRSsPZpvUlz-0) both variants (light & dark) are using 50 variants. Not sure, but probably a design bug. Maybe @mariocnovoa can take a look.

Reply:
@J-Son89 As @Parveshdhull mentioned, I followed the Figma designs. But I agree that with you both that this is probably a bug, so I updated it as you described. @mariocnovoa: if Figma is indeed correct, let me know, and I will revert this to use the same colors for both light and dark themes.